orchestra/static/orchestra/project-management/project-vis.es6.js

```diff
@@ -51,12 +51,12 @@ export default function projectVis (
         .attr('class', 'x label')
 
       scope.$on('orchestra:projectManagement:dataUpdate', vis.draw)
-      dataService.ready.then(function () {
-        if (projectId) {
-          dataService.changeProject(projectId)
-          visUtils.parentContainer.node().scrollLeft = 100
-        }
-      })
+      if (projectId) {
+        dataService.changeProject(projectId)
+        visUtils.parentContainer.node().scrollLeft = 100
+      } else {
+        dataService.ready
```

Comment:
do you have to return this or something? i don't get what this line does on its own.